specification/purview/resource-manager/Microsoft.Purview/stable/2021-07-01/purview.json

```diff
@@ -0,0 +1,1958 @@
+      "type": "object",
+      "properties": {
+        "capacity": {
+          "format": "int32",
+          "description": "Gets or sets the sku capacity. Possible values include: 4, 16",
+          "type": "integer"
+        },
+        "name": {
+          "description": "Gets or sets the sku name.",
+          "enum": [
+            "Standard"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "name",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "AccountProperties": {
+      "description": "The account properties",
+      "type": "object",
+      "properties": {
+        "cloudConnectors": {
+          "$ref": "#/definitions/CloudConnectors",
+          "description": "Cloud connectors.\r\nExternal cloud identifier used as part of scanning configuration."
+        },
+        "createdAt": {
+          "format": "date-time",
+          "description": "Gets the time at which the entity was created.",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdBy": {
+          "description": "Gets the creator of the entity.",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdByObjectId": {
+          "description": "Gets the creators of the entity's object id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "endpoints": {
+          "description": "The URIs that are the public endpoints of the account.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/AccountEndpoints"
+            }
+          ],
+          "readOnly": true
+        },
+        "friendlyName": {
+          "description": "Gets or sets the friendly name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "managedResourceGroupName": {
+          "description": "Gets or sets the managed resource group name",
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "managedResources": {
+          "description": "Gets the resource identifiers of the managed resources.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ManagedResources"
+            }
+          ],
+          "readOnly": true
+        },
+        "privateEndpointConnections": {
+          "description": "Gets the private endpoint connections information.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnection"
+          },
+          "readOnly": true
+        },
+        "provisioningState": {
+          "description": "Gets or sets the state of the provisioning.",
+          "enum": [
+            "Unknown",
+            "Creating",
+            "Moving",
+            "Deleting",
+            "SoftDeleting",
+            "SoftDeleted",
+            "Failed",
+            "Succeeded"
```

Comment:
Provisioning state enum list should also have "Canceled" as one of the states. [https://github.com/Azure/azure-resource-manager-rpc/blob/69e229f04cb6b91a82c97fc6f61d56055107c447/v1.0/async-api-reference.md#provisioningstate-property](https://github.com/Azure/azure-resource-manager-rpc/blob/69e229f04cb6b91a82c97fc6f61d56055107c447/v1.0/async-api-reference.md#provisioningstate-property)

Reply:
Right now, we don't support Canceled state but will definitely track as a work item post GA.

Reply:
Done.